app/models/user.rb

```diff
@@ -7,8 +7,12 @@ class User < ActiveRecord::Base
   validates :first_name, :last_name, presence: true
   validates :username, presence: true, uniqueness: true
   validates :email, email_format: { message: " is not in a valid format" }
-  validates :phone_number, format: { with:/\A\d+\Z/, message: "can only contain numbers" }, length: { is: 10, message: "must be 10 numbers in length" }, allow_blank: true
-  validates :password, length: { minimum: 8, message: "must have a minimum of 8 characters"}
+  validates :phone_number, format: { with:/\A\d+\Z/,
+            message: "can only contain numbers" },
+            length: { is: 10, message: "must be 10 numbers in length" },
+            allow_blank: true
+  validates :password, length: { minimum: 8,
+            message: "must have a minimum of 8 characters"}
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside } missing.